packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -102,6 +102,19 @@ export interface RemoteVideoTileMenuOptions {
    */
   isHidden?: boolean;
 }
+
+/* @conditional-compile-remove(click-to-call) */
+/**
+ * Options for the local video tile in the Call composite.
+ *
+ * @beta
+ */
+export interface LocalVideoTileOptions {
+  /**
+   * Position of the local video tile. If unset will render the local tile in the floating local position.
```

Comment:
Can we somehow make clear this only affects the call screen and not the config page nor the PiPiP

Reply:
yup I will update the comment! thanks for pointing this out.